tests/moe/test_alphamoe_sm100.py

```
@@ -0,0 +1,451 @@
+        _assert_order_tolerant(out, expected, abs_sum, label)
+
+
+def test_alphamoe_sm100_guard_skips_blocks_past_plan_extent():
+    """Blocks past num_tokens_post_padded must not touch out.
+
+    The plan buffers are worst-case sized and garbage-filled past the valid
+    extent (pair 0 / expert 0), so any block ignoring the guard adds a
+    visible spurious contribution to token 0.
+    """
+    _skip_if_not_sm100_family()
+    label, m, n, k, num_experts, top_k, block_m, shared, bal, scaling, seed = (
```

Ruff (0.16.0)

[warning] Unpacked variable `label` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)